be/test/exprs/agg/aggregate_test.cpp

```diff
@@ -1394,6 +1394,63 @@ TEST_F(AggregateTest, test_percentile_cont) {
     ASSERT_EQ(3, result_column->get_data()[0]);
 }
 
+TEST_F(AggregateTest, test_percentile_cont_non_empty_grid_serialize) {
```

Comment:
why not use sql-test to add more tests? That way is much easier.

Reply:
> why not use sql-test to add more tests? That way is much easier.

Sorry, I just failed to find the usage of sql-test, Could you please give me an example?

Reply:
> > why not use sql-test to add more tests? That way is much easier.
> 
> Sorry, I just failed to find the usage of sql-test, Could you please give me an example?

Already founded, never mind